base/stats.go

```
@@ -0,0 +1,747 @@
+package base
+
+import (
+	"expvar"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	StatViewFormat = "%s.%s"
+)
+
+type SgwStats struct {
+	GlobalStats     GlobalStat          `json:"global"`
+	DbStats         map[string]*DbStats `json:"per_db"`
+	ReplicatorStats *ReplicatorStats    `json:"per_replication"`
+
+	initDBStats               sync.Once
+	registeredReplicatorStats sync.Once
+}
+
+func (s *SgwStats) String() string {
+	bytes, _ := JSONMarshal(s)
```

Why is it safe to ignore the error here?  Seems like something that should log info at minimum.

---

Added an error log and returning empty json obj